### ISSUE TITLE
method send() not found for MidiOutputConnection

### ISSUE THREAD

**Issue by askask** [Boddlnagg/midir#12]
When I try to use the send() method to send MIDI data, the following error occurs
```
src/main.rs:62:31: 62:50 error: no method named `send` found for type `midir::alsa::MidiOutputConnection` in the current scope
src/main.rs:62                 try!(conn_out.send(&[144, 60, 1]));
                                             ^~~~~~~~~~~~~~~~~~~
```
This also happened when I copied the included test.rs to a new project and run it there, but not when I executed it with `cargo run --example test` in the midir project.
I used rustc 1.5.0-nightly (20a6938c0 2015-10-16).

**Comment by Boddlnagg** [comment]
Thanks for the report, I will have a look at this tomorrow!

**Comment by Boddlnagg** [comment]
This should be fixed in version 0.1.2, which I just uploaded to [crates.io](https://crates.io/crates/midir). Can you please confirm this?

**Comment by askask** [comment]
It works for me, thanks.
